release_notes/368-release-notes.adoc

```diff
@@ -81,23 +82,31 @@ In previous versions of {product-title}, the `write` permission for the `APIToke
 Beginning with {product-title-short} 3.68, Red Hat has updated the image version number convention for the `Scanner`, `Scanner DB`, and `Collector` images. The version numbers for these images now match the version numbers for the `Main` image.
 ====
 
+[IMPORTANT]
+====
+Due to the change in image repository names in `registry.redhat.io`, customers who mirror ACS
+   image repositories should verify their mirroring is set up to mirror images from these
+   new locations.
+====
+
 |===
 | Image | Description | Current version
 
 | Main
 | Includes Central, Sensor, Admission Controller, and Compliance.
 Also includes `roxctl` for use in continuous integration (CI) systems.
-| registry.redhat.io/advanced-cluster-security/rhacs-main-rhel8:{rhacs-version}
+| `registry.redhat.io/advanced-cluster-security/rhacs-main-rhel8:3.68.0`
```

Comment:
We should use concrete version `3.68.0` instead of placeholder `{rhacs-version}` here because it would be substituted with `3.69.0`, `3.70.0`, and so on in the future releases.

Reply:
@msugakov - thats an variable calling the common attributes. Its so we have to only update the version in one place and can use that everywhere else.

Reply:
@IAmATeaPot418 I think this is fine since it's on the release note for `3.68`. I.e. it shouldn't bump to `3.69` on the next release

Reply:
Yes, hardcoding this is the right thing to do.

Reply:
Yup you're right good call.